src/react/src/components/FacilityDetailsClaimFlag.jsx

```diff
@@ -66,7 +66,11 @@ const FacilityDetailsClaimFlag = ({
     const backgroundColor = getBackgroundColor(isClaimed, isPending);
     const claimFacilityLink = makeClaimFacilityLink(osId);
     return (
-        <div className={classes.root} style={{ backgroundColor }}>
+        <div
+            className={classes.root}
+            style={{ backgroundColor }}
+            data-testid="cliam-banner"
```

Comment:
Please correct this name to `claim-banner` and fix this name in associated test cases.

Reply:
Thanks! Done.